spec/factories.rb

```diff
@@ -0,0 +1,20 @@
+
+include ActionDispatch::TestProcess
+
+FactoryGirl.define do 
+    
+
+  factory :hipstergram do
+    image { fixture_file_upload(Rails.root.join('spec', 'photos', 'test.jpg'), 'image/jpg') }
+  end
+
+
+  factory :hipster do
+    email "test@test.com"
+    password 12345678
+    password_confirmation 12345678
```

Comment:
Style/NumericLiterals: Separate every 3 digits in the integer portion of a number with underscores(_). (https://github.com/bbatsov/ruby-style-guide#underscores-in-numerics)